transport/wsn/pom.xml

```diff
@@ -150,6 +150,12 @@
             <version>2.0.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.kaazing</groupId>
+            <artifactId>specification.x-kaazing-idle-timeout</artifactId>
+            <version>develop-SNAPSHOT</version>
```

Comment:
TODO: To be changed to the actual released version after this project has been released.